### DISCUSSION TITLE
View QRCode's via HTTPS?

### DISCUSSION THREAD

**Discussion by MrDevanWright** [YOURLS/YOURLS#3811] [Plugins]
I have the Example QRCode plugin installed, and it works great! However, when attempting to view the QR code via HTTPS it doesn't work. It only shows the QR code without a protocol or with HTTP. Any idea what I'm doing wrong or is this a limitation of the Example Plugin?

**Comment by dgw** [comment]
You can't load anything from YOURLS "without a protocol"… [The example plugin in question](https://yourls.org/docs/development/examples/qrcode) doesn't care about HTTP or HTTPs.

I honestly don't have any idea what you might be doing wrong, as you haven't provided any information about your environment/setup. Differences in behavior when loading YOURLS via HTTP vs. HTTPS could be caused by a server configuration mistake, or maybe your browser has cached something it shouldn't?

Try a different browser if you haven't. Use [devtools](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Tools_and_setup/What_are_browser_developer_tools) to monitor Network activity while accessing one of these broken QR links over HTTP and HTTPS, or check with [cURL](https://curl.se/) (if you're on a Linux box, it's probably already installed). See if there's an obvious difference in headers.

**Reply by MrDevanWright** [reply]
> maybe your browser has cached something it shouldn't

Probably, as the issues I was experiencing are no longer happening.
